apps/passport-server/src/services/telegramService.ts

```diff
@@ -358,7 +361,31 @@ export class TelegramService {
       });
     });
 
-    this.bot.on(":forum_topic_created", async (ctx) => {
+    // Edge case logic to handle routing people between bots
+    if (this.anonBotExists()) {
+      this.authBot.command("anonsend", async (ctx) => {
+        await ctx.reply(
+          `Command only available here: ${ctx.session.anonBotURL}?start=anonsend`
```

Comment:
Perhaps a friendlier message like "Please message Zuraffe to send anonymous messages 😎🦒"